src/forwardBeamPipeBrazil.cpp

```diff
@@ -48,6 +48,11 @@ static Ref_t create_detector(Detector& det, xml_h e, SensitiveDetector /* sens *
       -0.0454486856; //This is the angle of the proton orbit from the end of B1APF to the beginning of B2PF
   double crossingAngle = -0.025; //relevant for the neutral cone
 
+  double b0PFCenter_z =
+      640.0 *
```

Comment:
Can we avoid hardcoding this? Looks like this can become a value read from xml. If it's the same meaning as in src/magnetVacuumFF.cpp, it would make sense to define a `<constant>` for it in `compact/defintions.xml`.

Reply:
this constant already defined in https://github.com/eic/epic/blob/e52d7c5f6313a1fad7163db0d6ed2fc076ac887a/compact/far_forward/definitions.xml#L79

I introduced it to the code in https://github.com/eic/epic/pull/788/commits/660d5cb9a3a88ebb8635a0ac4586c1697a224dd7